lib/node_modules/@stdlib/stats/base/dstdev/manifest.json

```
@@ -27,55 +28,70 @@
   "confs": [
     {
       "task": "build",
+      "wasm": false,
       "src": [
-        "./src/dstdev.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
-      ],
+      "libraries": [],
       "libpath": [],
       "dependencies": [
+        "@stdlib/blas/base/shared",
         "@stdlib/stats/strided/dstdevpn",
         "@stdlib/napi/export",
         "@stdlib/napi/argv",
         "@stdlib/napi/argv-int64",
```

```suggestion
        "@stdlib/napi/argv-int64",
        "@stdlib/napi/argv-double",
```

Not clear why `argv-double` is being removed. Beware of copy-paste mistakes.